simple-mpc.el

```diff
@@ -123,36 +123,5 @@ command."
   (simple-mpc-call-mpc nil (list "load" playlist-name))
   (simple-mpc-maybe-refresh-playlist))
 
-;;;###autoload
-(defun simple-mpc (&optional ignore-auto noconfirm)
-  "Start simple-mpc.
-
-IGNORE-AUTO and NOCONFIRM are passed by `revert-buffer'."
-  (interactive)
-  (let ((buf (get-buffer-create simple-mpc-main-buffer-name)))
-    (with-current-buffer buf
-      (read-only-mode -1)
-      (erase-buffer)
-      (insert (propertize "* simple-mpc *\n\n"
-                          'face 'simple-mpc-main-name)
-              (propertize "   * controls\n" 'face 'simple-mpc-main-headers)
-              "      * [t]oggle\n"
-              "      * [n]ext track\n"
-              "      * [p]revious track\n"
-              "      * seek [f]orward\n"
-              "      * seek [b]ackward\n"
-              "      * increase [V]olume\n"
-              "      * decrease [v]olume\n"
-              (propertize "\n   * playlist\n" 'face 'simple-mpc-main-headers)
-              "      * view [c]urrent playlist\n"
-              "      * [C]lear current playlist\n"
-              "      * [S]huffle playlist\n"
-              "      * [l]oad playlist\n"
-              "      * [s]earch database\n"
-              (propertize "\n   * misc\n" 'face 'simple-mpc-main-headers)
-              "      * [q]uit")
-      (simple-mpc-mode) ; start major mode
-      (switch-to-buffer buf))))
-
 (provide 'simple-mpc)
```

Comment:
We should move this `provide` as well. According to [the documentation](https://www.gnu.org/software/emacs/manual/html_node/elisp/Named-Features.html) the file that defines the symbol should provide it.

Reply:
Ah, good catch; thanks! I'll re-read the code changes and also the docs :-)

Reply:
I might rethink my solution a bit to see if there's a better route; I'll try to do some tonight.